cmd/fleetctl/fleetctl/generate_gitops.go

```diff
@@ -566,8 +572,14 @@ func generateFilename(name string) string {
 			return unicode.ToLower(r)
 		case unicode.IsSpace(r):
 			return '-'
+		// replace common footguns with unique letters.
+		case slices.Contains(footguns, r):
+			return rune('a' + slices.Index(footguns, r))
```

Comment:
IMO we shouldn't be allowing these in team names in the first place, but to be extra safe and attempt to have unique filenames, we'll just translate problematic characters into letters.